js/bundle.js

```diff
@@ -128,7 +128,7 @@ function row(field, title, formatter, tag) {
             .enter()
             .append('td')
             .attr('class', function(d) {
-                return 'version_cell ' + pClass(tag ? d.tags[field] : d[field]);
+                return 'version_cell ' + (d.visible ? pClass(tag ? d.tags[field] : d[field]) : 'removed');
```

Comment:
I'd rather not have the entire column use `removed` when it's deleted. If anything, let's add a row above the tags section for the `visible` boolean.

Reply:
Add a new row for `visible` may be a bit redundant since it's usually only be false at last version.
How about just use `removed` on the `version` cell? Please look at my updated at https://aleung.github.io/osm-deep-history/#/node/1 
Perhaps you need to clear recent browser cache to see the change.